modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -2,72 +2,213 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: scheduler-xml-reference.adoc
 
-The Scheduler component enables you to trigger a flow when a time-based condition is met. You can configure it to be triggered at a regular interval or give it a more flexible cron expression. For example, a Scheduler might trigger an event to start a flow every 5 seconds.
+The Scheduler component is a Mule message source that triggers the execution of a flow based on a time-based condition and creates a new Mule message for each execution of the flow. For example, a Scheduler might start a new instance of the flow and create a new Mule message for that flow every 5 seconds or every day at 12:00 in a given time zone.
 
-Consider the following when you implement a Scheduler in your Mule application:
+Consider the following when adding a Scheduler to your Mule application:
 
 * Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
-* The `disallowConcurrentExecution` property enables you to prevent the Scheduler from re-triggering a flow until it finishes processing. See xref:scheduler-xml-reference#scheduler-properties[<scheduler> Properties].
-* If xref:execution-engine.adoc#backpressure[back-pressure] occurs because no resources are available at the time of the scheduled trigger, Mule skips the trigger.
++
+To set a time zone for the Scheduler, see `timeZone` in <<props_cron_expressions>>.
+* The `disallowConcurrentExecution` property enables you to prevent the Scheduler from triggering the next execution of a flow while the previously executed flow is still running.
```

Comment:
```suggestion
* The `disallowConcurrentExecution` property enables you to prevent the Scheduler from triggering the execution of the flow while a previously triggered execution is still running.
```